bluemira/equilibria/plotting.py

```diff
@@ -100,18 +101,35 @@ class Plotter:
     Utility plotter abstract object
     """
 
-    def __init__(self, ax=None, **kwargs):
+    def __init__(self, ax=None, subplots=False, **kwargs):
```

Comment:
This will fix your linter issues 
```suggestion
    def __init__(self, ax=None, *, subplots=False, **kwargs):
```